src/class-rest-api-endpoints.php

```diff
@@ -25,15 +28,16 @@ public function __construct() {
 	 *
 	 * @return void
 	 */
-	public function register_endpoints() {
+	public function register_endpoints(): void {
 		register_rest_route(
 			'wp-newsletter-builder/v1',
 			'/lists/',
 			[
 				'methods'             => 'GET',
 				'callback'            => [ $this, 'get_lists' ],
 				'permission_callback' => function () {
-					return current_user_can( 'edit_posts' );
+					return true;
+					// return current_user_can( 'edit_posts' ); TODO.
```

Comment:
Here is my situation. This is a work in progress. We are waiting on client instructions before continuing development on the email integration. I would like to get these changes merged into develop so that Greg's work is using all of the same dependencies and some bug fixes here are included. We will be making new issues to handle these, but are waiting on client instructions before doing so, however I would prefer this doesn't stop all progress. This plugin is not being used on any sites yet. While this is not the standard practice, I think it makes sense to get this merged ahead of completing this work or removing these code comments. Open to your thoughts though, I am not fixated on this. Maybe I need to cherry pick commits or keep working on this, but it will need to be ticketed as PD if we do that, if we wait until we get client instructions, we can easily make it a part of our billable work.